stl/src/vector_algorithms.cpp

```diff
@@ -3845,6 +3845,208 @@ __declspec(noalias) void __stdcall __std_replace_8(
 
 } // extern "C"
 
+namespace {
+    template <class _Ty>
+    void* _Remove_fallback(void* const _First, void* const _Last, void* const _Out, const _Ty _Val) noexcept {
+        _Ty* _Src  = reinterpret_cast<_Ty*>(_First);
+        _Ty* _Dest = reinterpret_cast<_Ty*>(_Out);
+
+        while (_Src != _Last) {
+            if (*_Src != _Val) {
+                *_Dest = *_Src;
+                ++_Dest;
+            }
+
+            ++_Src;
+        }
+
+        return _Dest;
+    }
+
+
+    template <size_t _Size_v, size_t _Size_h>
+    constexpr auto _Make_remove_tables(const unsigned _Mul, const unsigned _Ew) {

```

Comment:
We conventionally say `unsigned int`, never `unsigned`. Occurs below.

Reply:
Let it be `unit32_t` then to avoid too much verbosity. 

Intrinsics code isn't portable, so saying `unsigned int` versus `uint32_t` does not actually convey any more information.